### ISSUE TITLE
Invoking an Action constant fails with 'Common Language Runtime detected an invalid program'

### ISSUE THREAD

**Issue by SteveWilkes** [dadhi/FastExpressionCompiler#156]
Hello! 

I've added a failing test in PR #155 - using an `InvocationExpression` to invoke an `Action` stored in a `ConstantExpression` fails with 'Common Language Runtime detected an invalid program'.

Cheers,

Steve

**Comment by dadhi** [comment]
Thanks and omg `Constant(1, typeof(object))`... sigh :(

**Comment by jogibear9988** [comment]
fixed

**Comment by SteveWilkes** [comment]
Fix works for me! Thanks :)

And is `Constant(1, typeof(object))`... bad...?

**Comment by dadhi** [comment]
Not really, just did not expect such a usage. Usually I am doing Convert. Anyway, interesting to see a new ways :)

**Comment by jogibear9988** [comment]
can be closed, is fixed
